indexinglanguage/src/main/java/com/yahoo/vespa/indexinglanguage/expressions/NormalizeExpression.java

```diff
@@ -81,7 +81,7 @@ public String toString() {
     public boolean equals(Object o) {
         if (!(o instanceof NormalizeExpression)) return false;
         NormalizeExpression other = (NormalizeExpression)o;
-        if (linguistics != other.linguistics) return false;
+        if (linguistics.getClass() != other.linguistics.getClass()) return false;
```

Comment:
Any reason why this differs from the one in NGramExpression ?
Are the classes from the same bundle, or will might they be from different model versions ?

Reply:
> Any reason why this differs from the one in NGramExpression ?

here linguistics must be present, but in ngram it can be null.

> Are the classes from the same bundle, or will might they be from different model versions ?

after more thinking I'm leaning towards just removing the check for equal linguistics; I think we have (or are planning on) some other mechanism for triggering reindexing when we upgrade Yell or make other changes to linguistics config.  I'm guessing we'll get the same class when they are from different model versions, but I'm not 100% sure.